packages/cli/src/lib/bundler/config.ts

```diff
@@ -175,18 +181,24 @@ export async function createConfig(
     );
   }
 
+  const rspack = useRspack
+    ? (require('@rspack/core') as typeof import('@rspack/core').rspack)
+    : undefined;
+  const bundler = useRspack ? (rspack as unknown as typeof webpack) : webpack;
```

Comment:
There's a couple of places where we need access to `require('@rspack/core')` for setting things up. I'm thinking that it would be worthwhile to refactor so that rather than passing around `useRspack: boolean`, we instead pass around an optional RSPack instance, which doubles as the flag, i.e. `rspack?: typeof import('@rspack/core').rspack`.